internal/command/publishreleaseartifacts.go

```diff
@@ -31,7 +31,7 @@ import (
 
 var CmdPublishReleaseArtifacts = &Command{
 	Name:  "publish-release-artifacts",
-	Short: "Publishes previously-created release artifacts to package managers etc",
+	Short: "Publish release artifacts to package managers.",
```

Comment:
I'd like there to be something to indicate that this shouldn't be doing a build.